index-v4.yaml

```diff
@@ -87,5 +87,11 @@ distributions:
     distribution_status: active
     distribution_type: ros1
     python_version: 3
+  rolling:
+    distribution: [rolling/distribution.yaml]
+    distribution_cache: http://repo.ros2.org/rosdistro_cache/rolling-cache.yaml.gz
+    distribution_status: rolling
```

Comment:
Do we need to update any infrastructure to handle this new `status` value?

Reply:
As far as I can recall most of the code leveraging distribution_status is doing so to exclude distributions with status `end-of-life`. It's been a while since I've grep'd through all the ros-infrastructure tools but I'll do so again before merging as a double-check.

Reply:
Yep looking through rosdep, rosdistro, rospkg, and bloom https://github.com/ros-infrastructure/rosdep/blob/d1784da592310c7ad8e649530200cdf957a87afc/src/rosdep2/sources_list.py#L502 is the only current usage of the distribution status that isn't in a test.